packaging_automation/citus_package.py

```diff
@@ -71,16 +85,21 @@ class PostgresVersionDockerImageType(Enum):
 
 
 def decode_os_and_release(platform_name: str) -> Tuple[str, str]:
-    parts = platform_name.split("/")
+    separator = "," if "," in platform_name else "/"
+    parts = platform_name.split(separator)
 
     if len(parts) == 0 or len(parts) > 2 or (len(parts) == 1 and parts[0] != "pgxn"):
-        raise ValueError("Platforms should have two parts divided by '/' or should be 'pgxn' ")
+        raise ValueError("Platforms should have two parts divided by '/',',' or should be 'pgxn' ")
     if len(parts) == 1 and parts[0] == "pgxn":
         os_name = "pgxn"
         os_release = ""
     else:
         os_name = parts[0]
         os_release = parts[1]
+        # When building packaging images oraclelinux and centos is used instead of el and ol
+        # Below two lines handles conversion betwwen them
+        os_name = os_name if os_name != "oraclelinux" else "ol"
+        os_name = os_name if os_name != "centos" else "el"
```

Comment:
```suggestion
        os_name = "ol" if os_name == "oraclelinux" else os_name
        os_name = "el" if os_name == "centos" else os_name
```

I think this one is easier to read and understand.